File: lib/models/garment_model.dart
import 'package:flutter/material.dart';

class GarmentModel {
  final String id;
  final String name;
  final String category;
  final String description;
  final Color color;
  final bool is3D;

  const GarmentModel({
    required this.id,
    required this.name,
    required this.category,
    required this.description,
    required this.color,
    this.is3D = true,
  });
}

File: unity-mvp/Assets/Scripts/GarmentModel.cs
using UnityEngine;

[CreateAssetMenu(fileName = "GarmentModel", menuName = "AR Clothing/Garment Model")]
public class GarmentModel : ScriptableObject
{
    public string garmentId;
    public string displayName;
    public string category;
    public string description;
    public Color accentColor = Color.white;
    public GameObject garmentPrefab;
}
